swagger.json

```diff
@@ -1641,7 +1641,87 @@
                 }
             }
         },
-      "/fleet/vehicles/locations": {
+        "/fleet/vehicles/{vehicle_id_or_external_id}": {
+            "get": {
+                "tags": [
+                    "Fleet"
+                ],
+                "summary": "/fleet/vehicles",
+                "description": "Gets a specific vehicle.",
+                "operationId": "getFleetVehicle",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "$ref": "#/parameters/vehicleIdOrExternalId"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The specified vehicle.",
+                        "schema": {
+                            "$ref": "#/definitions/FleetVehicleResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": ["Fleet"],
+                "summary": "/fleet/vehicles",
+                "description": "Updates the specified vehicle",
```

Comment:
Could we also elaborate on the patch semantics? We should describe that users should use https://tools.ietf.org/html/rfc7396 

This includes the appropriate `Content-Type: application/merge-patch+json` and JSON Patch body.

Reply:
Would it be correct to say that this endpoint consumes like
```                
"consumes": [
    "application/json",
    "application/merge-patch+json"
 ],
```
for PATCH, or only `application/merge-patch+json`?